### ISSUE TITLE
Integrate hs images

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-web#177]
The images are stored in Repro\Multispektral

**Comment by thvitt** [comment]
Is this different material than #204? If so, how does it relate to the current contents? should this be integrated in the info section only or do we need UI?

**Comment by gerritbruening** [comment]
> Is this different material than #204

not really. The images could be placed where all the other images are, and they could be pointed to from the same XML transcripts.

> should this be integrated in the info section

Once the hs images are pointed to from the XML transcripts, it would be great to generate a list of pages from which we have hs images. Entries in this list could simply link to the facsimile view of the corresponding page.

**Comment by thvitt** [comment]
ah, fine. We probably need to place some classification somewhere (also for #204)? IIRC the other images are just additional values for `pb/@facs`?

**Comment by gerritbruening** [comment]
Not quite: they are referenced via facsimile in graphic from the documentary transcripts.
We could have an @ type in the facsimile element.

**Comment by gerritbruening** [comment]
Maybe there is some, we would have to ask @wissenbach.

**Comment by gerritbruening** [comment]
@wissenbach, did you already see this:
https://www.manuscript-cultures.uni-hamburg.de/lab_case_11_e.html

**Comment by gerritbruening** [comment]
Sorry that I return on this so late.
The information you give is valuable and worth to keep. Thank you for this!
I see your point, but wouldn't it still be worth to store the images in a lab-like area of the edition? We have more material that we cannot really integrate in the main parts of the edition ([Schillemeit, direction books, drawings](https://github.com/faustedition/faust-web/issues/228)).

**Comment by wissenbach** [comment]
Thanks for the link, @gerritbruening. That sample result looks good. Our results did also separate the different layers visually to some degree, but did not really yield any new transcriptions. 
![500nm](https://user-images.githubusercontent.com/650995/41534909-95f56b10-7300-11e8-934d-f11ff2b44f19.jpg)
![ridge-regression](https://user-images.githubusercontent.com/650995/41534922-a3bb6600-7300-11e8-89eb-c9a6263fede3.jpg)
I'm sure that I didn't exhaust the full potential of the image material we have, so I think putting them online is a good idea.

The easiest way to put up these materials (including Schillemeit etc.) would be as a downloadable archive.

**Comment by gerritbruening** [comment]
The manuscript cultures sample is from this manuscript here:
http://dev.faustedition.net/document?sigil=2_II_H.47&page=2&view=facsimile_document

> Our results did also separate the different layers visually to some degree, but did not really yield any new transcriptions.

This holds also true for the example from the manuscript cultures case, so there is no reason to hide one's light under a bushel.
